sdk-core-protos/src/lib.rs

```diff
@@ -258,75 +259,7 @@ pub mod coresdk {
     }
 
     pub mod external_data {
-        use prost_types::{Duration, Timestamp};
-        use serde::{Deserialize, Deserializer, Serialize, Serializer};
         tonic::include_proto!("coresdk.external_data");
-
-        // Buncha hullaballoo because prost types aren't serde compat.
-        // See https://github.com/tokio-rs/prost/issues/75 which hilariously Chad opened ages ago
```

Comment:
Nice! I'm assuming histories are still compatible, right?

Reply:
This is what using prost-wkt gives us. I would prefer pbjson but it has this problem https://github.com/influxdata/pbjson/issues/23

Perhaps I'll just revert this for now and keep my manual solution rather than adding all these deps.

Reply:
Really I just want this upstreamed into prost